.evergreen/config.yml

```diff
@@ -1320,14 +1349,47 @@ tasks:
         script: |-
           set -o errexit
           ${PREPARE_SHELL}
-          git add .
-          git commit -m "add files"
-          export AZUREOIDC_DRIVERS_TAR_FILE=/tmp/mongo-rust-driver.tgz
-          git archive -o $AZUREOIDC_DRIVERS_TAR_FILE HEAD
-          export AZUREOIDC_TEST_CMD="PROJECT_DIRECTORY='.' ./.evergreen/install-dependencies.sh rust\
-                                     && PROJECT_DIRECTORY='.' .evergreen/install-dependencies.sh junit-dependencies\
-                                     && PROJECT_DIRECTORY='.' OIDC_ENV=azure OIDC=oidc  ./.evergreen/run-mongodb-oidc-test.sh"
-          bash $DRIVERS_TOOLS/.evergreen/auth_oidc/azure/run-driver-test.sh
+           ./.evergreen/install-dependencies.sh rust
+          source .cargo/env
+          export AZUREOIDC_DRIVERS_TAR_FILE=/tmp/mongo-rust-driver.tar
+          rustup default stable
+          export RUSTFLAGS="-C target-feature=+crt-static"
```

Comment:
what is this needed for?

Reply:
The VM has a different version of glibc

Reply:
The longer answer is the GCP VM is too slow to compile the test (it takes over 3 hours, assuming it can ever finish), so the strategy I originally used for Azure of compiling the test on the VM didn't work. I managed to build this and get it on the VM just to find out the wrong version of glibc is on the VM. This makes it work.